### ISSUE TITLE
Error when changing route from view with a ak-drop-down-list

### ISSUE THREAD

**Issue by patrek** [aurelia-ui-toolkits/aurelia-kendoui-bridge#452]
When I move away from a route where I have a ak-drop-down-list, I get the following error:

```
kendo.dropdownlist.js:168 Uncaught (in promise) TypeError: Cannot read property 'off' of null
    at r.extend.destroy (http://localhost:8282/jspm_packages/github/kendo-labs/bower-kendo-ui@2016.1.322/js/kendo.dropdownlist.min.js:25:2619)
    at WidgetBase.destroy (http://localhost:8282/jspm_packages/npm/aurelia-kendoui-bridge@0.6.4/common/widget-base.js:165:16)
    at DropDownList.detached (http://localhost:8282/jspm_packages/npm/aurelia-kendoui-bridge@0.6.4/dropdownlist/dropdownlist.js:78:23)
    at Controller.detached (http://localhost:8282/jspm_packages/npm/aurelia-templating@1.0.0-beta.1.2.1/aurelia-templating.js:2894:26)
    at View.detached (http://localhost:8282/jspm_packages/npm/aurelia-templating@1.0.0-beta.1.2.1/aurelia-templating.js:1044:26)
    at removeAction (http://localhost:8282/jspm_packages/npm/aurelia-templating@1.0.0-beta.1.2.1/aurelia-templating.js:1366:25)
    at ViewSlot.removeAll (http://localhost:8282/jspm_packages/npm/aurelia-templating@1.0.0-beta.1.2.1/aurelia-templating.js:1385:7)
    at after (http://localhost:8282/jspm_packages/npm/aurelia-templating-router@1.0.0-beta.1.2.0/router-view.js:89:39)
    at work (http://localhost:8282/jspm_packages/npm/aurelia-templating-router@1.0.0-beta.1.2.0/router-view.js:165:9)
    at eval (http://localhost:8282/jspm_packages/npm/aurelia-templating-router@1.0.0-beta.1.2.0/router-view.js:182:11)
```

both in 0.6.2 and 0.6.4 versions

**Comment by JeroenVinke** [comment]
Didn't work, I'm now thinking that [this](https://github.com/telerik/kendo-ui-core/blob/master/src/kendo.angular.js#L536-L546) me be the best way to destroy controls

**Comment by JeroenVinke** [comment]
I just released 0.7.1 which should solve the issue

**Comment by JeroenVinke** [comment]
@patrek can you confirm 0.7.1 resolved the issue?

**Comment by patrek** [comment]
Yes, 0.7.1 fixed it for us. Thanks.

**Comment by JeroenVinke** [comment]
unfortunately, this issue is back

**Comment by JeroenVinke** [comment]
should be fixed in 1.5.3
